openstates/ar/committees.py

```diff
@@ -20,8 +20,14 @@ def scrape(self, chamber=None, session=None):
         if session is None:
             session = self.latest_session()
             self.info('no session specified, using %s', session)
+
+        if session == '2018':
```

Comment:
It may make more sense to turn off the committee scraper, if there're no good committees during the fiscal session, rather than hardcode to using 2017 committees (I can see us forgetting to switch back, come 2018 Regular).

Or if this is an every-fiscal-session thing, then maybe do something like `sess1 = str(int(session[:4]) - 1) if session.endswith('F') else session[:4]`.